src/main/java/uk/ac/ebi/embl/api/validation/fixer/entry/QualifierRemovalFix.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright 2012 EMBL-EBI, Hinxton outstation
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package uk.ac.ebi.embl.api.validation.fixer.entry;
+
+import uk.ac.ebi.embl.api.entry.Entry;
+import uk.ac.ebi.embl.api.entry.feature.Feature;
+import uk.ac.ebi.embl.api.entry.qualifier.Qualifier;
+import uk.ac.ebi.embl.api.validation.Severity;
+import uk.ac.ebi.embl.api.validation.ValidationResult;
+import uk.ac.ebi.embl.api.validation.ValidationScope;
+import uk.ac.ebi.embl.api.validation.annotation.Description;
+import uk.ac.ebi.embl.api.validation.annotation.ExcludeScope;
+import uk.ac.ebi.embl.api.validation.check.entry.EntryValidationCheck;
+
+import java.util.List;
+
+@Description("Qualifier \"{0}\" removed from feature \"{1}\"")
+@ExcludeScope(validationScope = {ValidationScope.NCBI , ValidationScope.NCBI_MASTER})
+public class QualifierRemovalFix extends EntryValidationCheck {
+	private final static String MESSAGE_ID = "RemoveCitationQualifierFix";
+	public QualifierRemovalFix() {
+	}
+
+	public ValidationResult check(Entry entry) {
```

Comment:
Just realized, will be better if its it   a Feature level check rather than Entry level e.g., ObsoleteFeatureFix.java or all the checks in this package
package uk.ac.ebi.embl.api.validation.fixer.feature;

Reply:
I can move the code to Feature level